guides/v2.0/cloud/configure/fastly-vcl-extend-timeout.md

```diff
@@ -23,17 +23,18 @@ The important code that sets the timeout is the `first_byte_timeout` value of 10
 This information is just the code portion for setting up your VCL. Use this information with [Custom Fastly VCL snippets]({{page.baseurl}}cloud/configure/cloud-vcl-custom-snippets.html).
 </div>
 
-## Create extendtimeout.vcl {#vcl}
+## Create extendtimeout.json {#vcl}
```

Comment:
According to [this Slack comment](https://magento.slack.com/archives/C1EAG6PU3/p1517418335000173), it's no longer necessary to create an `extendtimeout.json` file. The latest version of the plugin supports setting it from the UI or CLI. @yyevgenii, how should we handle this?

Reply:
We're going to go ahead and merge this because it'll immediately help customers. @yyevgenii, we can always address this issue later.

Reply:
@jeff-matthews This feature is available in the latest release of fastly/magento2 module. I'm going to check this functionality and add some notes to the article: that on the newer version users can manage this timeout through Magento admin backend. Then we can remove the fastly-vcl-extend-timeout article at all.